scripts/new_project.kts

```diff
@@ -1,25 +1,55 @@
+#!/usr/bin/env kscript
```

Comment:
Do we still need this? If I'm not mistaken we no longer need this, unless we import dependencies